documentation/pdb_client_tools.markdown

```diff
@@ -6,11 +6,15 @@ layout: default
 [installpuppet]: /puppet/latest/reference/install_pre.html
 [repos]: /puppet/latest/reference/puppet_collections.html
 [export]: ./anonymization.html
+[installpeclienttools]: /pe/latest/install_pe_client_tools.html#non-managed-workstation-installation
 
 # PuppetDB CLI
 
 ## Installation
 
+If you are installing `pe-client-tools` please see
+[the pe-client-tools installation instructions][installpeclienttools].
+
```

Comment:
Do you want to specify here that you can install the client tools on a managed or non-managed workstation?

It's all covered in the installation docs: 

https://docs.puppet.com/pe/latest/install_pe_client_tools.html